### ISSUE TITLE
Two test cases are failing in 2.1 branch after merge from master

### ISSUE THREAD

**Issue by cloudysail** [TresAmigosSD/SMV#1113]
There are two test cases failing in 2.1 branch after my merge from master
[error]     org.tresamigos.smv.SmvJdbcTest
[error]     org.tresamigos.smv.CsvTest

Need to figure out.

**Comment by laneb** [comment]
Done. Problem with the CSV test was that when you merged master onto 2.1 you changed the line
```
 val res = df.map(r => (r.getString(0), r.getInt(1))).collect.mkString(",")
```
to 
```
 val res = df.map(r => (r(0), r(1))).collect.mkString(",")
```
which was not valid in context. I had trouble tracking down the cause of the error with the JdbcTest, I reset your changes and did my own merge and didn't run into any issues like that. Fyi I force-pushed my copy of the branch to clean up the commit history, so you'll need to reset to before your merge attempt and pull my commit.

**Comment by cloudysail** [comment]
Thanks @laneb .

For the JdbcTest, I figured out it was because I missed the following line in SmvDataset->publishThroughJDBC
```
connectionProperties.put("driver", app.smvConfig.jdbcDriver)
```

It seems there are many 2.1 specific codes we need to remember, so that whenever conflict happens during merge from the master, we need to carefully manually solve those conflict.
